sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/DevModeMysqlDevServiceUserExperienceIT.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.ts.sqldb.sqlapp;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.github.dockerjava.api.model.Image;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.services.DevModeQuarkusApplication;
+import io.quarkus.test.utils.SocketUtils;
+
+@QuarkusScenario
+@DisabledOnNative
```

Comment:
We don't need to add this annotation any longer (they are not executed for tests with DevMode).